src/main/java/de/dennisguse/opentracks/ShowErrorActivity.java

```diff
@@ -0,0 +1,86 @@
+package de.dennisguse.opentracks;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+
+import de.dennisguse.opentracks.databinding.ActivityShowErrorBinding;
+
+public class ShowErrorActivity extends AppCompatActivity {
+
+    public static final String EXTRA_ERROR_TEXT = "error";
+
+    private static final String TAG = ShowErrorActivity.class.getSimpleName();
+
+    private ActivityShowErrorBinding binding;
+
+    @Override
+    public void onCreate(final Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        binding = ActivityShowErrorBinding.inflate(getLayoutInflater());
+        setContentView(binding.getRoot());
+
+        binding.textViewError.setText(getIntent().getStringExtra(EXTRA_ERROR_TEXT));
+
+        binding.bottomAppBarLayout.bottomAppBarTitle.setText(createErrorTitle());
+        setSupportActionBar(binding.bottomAppBarLayout.bottomAppBar);
+    }
+
+    private String createErrorTitle() {
+        return String.format(getString(R.string.error_crash_title), getString(R.string.app_name));
+    }
+
+    private void reportBug() {
+        Uri uriUrl;
+        try {
+            uriUrl = Uri.parse(
+                    String.format(
+                            getString(R.string.report_issue_link),
+                            URLEncoder.encode(binding.textViewError.getText().toString(), StandardCharsets.UTF_8.toString())
+                    )
+            );
+        } catch (final UnsupportedEncodingException e) {
+            Log.e(TAG, "Can't encode error report", e);
```

Comment:
Do we need user feedback for this case?
Or is this implicit?

Reply:
I think this can't happen as UTF-8 is available everywhere, but the Exception has to be caught. I can remove the `Log.e()` and rename the variable to `ignored`, if you like.